subset/security/tlsfaux/generate_certs.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+
+filepath = 'certs/'
+
+command_key = 'openssl req -nodes -newkey rsa:2048 -keyout '+filepath+'server.key -out '+filepath+'server.csr -subj "/C=GB/ST=London/L=KingsX/O=ExcelRedstone/OU=Software/CN=127.0.0.1"'
+command_cert ='openssl x509 -req -days 365 -in '+filepath+'server.csr -signkey '+filepath+'server.key -out '+filepath+'server.crt'
+
+def send_shell_command(command):
```

Comment:
send? Where is this being sent?  I think you mean run_?

Reply:
changed to run shell command